### PR TITLE
Updated docs icons to use the newly created placeholder icon

### PR DESCRIPTION
**Pull request by patricklafrance**
## Summary

End of the day and was too tired to start something new so I've decided to update the docs example icons to use the newly created placeholder icon!

### CODE REVIEW

**Comment by fraincs** [comment]
image

I would have expected a placeholder icon here

File: packages/components/src/listbox/docs/Listbox.stories.mdx [hunk 1]
```diff
@@ -135,18 +135,22 @@ However, think twice before adding *end* icons, *start* icons should be your go
...
                 </IconList>
             </Item>
             <Item key="jupiter">
                 <Text>Jupiter</Text>
-                <HelpMajorIcon slot="end-icon" />
+                <PlaceholderMajorIcon slot="end-icon" />
             </Item>
             <Item key="mars">
                 <Text>Mars</Text>
-                <LightbulbMajorIcon slot="end-icon" />
```
Comment (fraincs) [inline packages/components/src/listbox/docs/Listbox.stories.mdx:149]: What is the reason for the change here?

**Review by fraincs — COMMENTED** [review]
nice job

**Comment by patricklafrance** [comment]
> image
> 
> I would have expected a placeholder icon here. Although this one open to debate as it might means that you can edit the input.

Yeah, wasn't sure, the pencil kind of made sense

File: packages/components/src/listbox/docs/Listbox.stories.mdx (hunk 1 above)
Comment (patricklafrance) [reply packages/components/src/listbox/docs/Listbox.stories.mdx:149]: Other list components docs example are all like this

**Review by fraincs — APPROVED** [review]
